docs/architecture/cx-3-2/edc/policy.definitions.md

```diff
@@ -0,0 +1,272 @@
+    "rightOperand": "vehicleDismantle"
+  }
+}
+```
+
+> ISSUE: What are the `rightOperand` values?
+
+## 2.3. Dismantler.allowedBrands
+
+***Not supported in 3.2**
+
+The `Dismantler.allowedBrands` subtype requires the dismantler to be certified for a set of brands. It is expressed
+using an `in` operator as
+follows:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "Dismantler.allowedBrands",
+    "operator": "in",
+    "rightOperand": [
+      "Brand A",
+      "Brand B"
+    ]
+  }
+}
+```
+
+The `rightOperand` is an array of type `string` containing one or more elements. The elements are evaluated using a
+logical OR.
+
+# 3. Framework Agreement Constraints
+
+Framework agreement constraints adhere to the following syntax:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.[type]-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+The `type` segment specifies the framework case agreement type. The version segment is optional.
+
+Framework agreement constraints may only be used as an ODRL `Permission`.
+
+> ISSUE: What are the `rightOperand` values?
+
+## 3.1. PCF
+
+The PCF framework agreement is expressed as:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.pcf-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+## 3.2. Sustainability
+
+The Sustainability framework agreement is expressed as:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.sustainability-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+## 3.3. Quality
+
+The Quality framework agreement is expressed as:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.quality-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+## 3.4. Resiliency
+
+The Resiliency framework agreement is expressed as:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.resiliency-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+## 3.5. Traceability
+
+The Traceability framework agreement is expressed as:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.traceability-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+## 3.6. Behavioral Twin
+
+The Behavior framework agreement is expressed as:
+
+```json
+{
+  "constraint": {
+    "leftOperand": "FrameworkAgreement.behavioraltwin-[version]",
+    "operator": "eq",
+    "rightOperand": "active"
+  }
+}
+```
+
+## 3.7. BPN
+
+> ISSUE: The Membership VC appears to contain the same information. Is the BPN VC needed?
+
+# 4. Policy Example
+
+The following is an `Offer` that requires the data user to be a certified dismantler for `Brand A` vehicles and to have
+an active signed traceability agreement:
+
+```json
+ {
+  "@context": [
+    "http://www.w3.org/ns/odrl.jsonld",
+    {
+      "cx": "https://w3id.org/tractusx/v0.0.1/ns/"
```

Comment:
nit: To avoid ambiguity, I would either embed the _entire_ context, or at least those parts that are used here (`Dismantler.allowedBrands`, `FrameworkAgreement.traceability`).

Reply:
right, but i'm saying that if people now the JSON-LD from the document, and stick it into some JSON-LD playground-like tool, because they want to see what it looks like, it should produce the correct results. It's only two lines.

Reply:
This will need to be linked externally, it cannot be embedded. I will explain offline.